Nowy test.py

```diff
@@ -1,2 +1,2 @@
-slownik = {"herbata" : "zwykla"}
+slownik = {"herbata" : "zielona"}
```

Comment:
Hej, tutaj powinna być czarna herbata według dokumentacji. Weź się ogarnij misiu!!